e2e/scenario/scenario_azurelinuxv2-chrony-restarts.go

```diff
@@ -0,0 +1,44 @@
+package scenario
+
+import (
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+)
+
+func (t *Template) azurelinuxv2SystemdChronyDropin() *Scenario {
+	return &Scenario{
+		Name:        "azurelinuxv2-systemd-dropin-for-chrony",
+		Description: "Tests that the systemd drop-in file for chrony is placed in the right place",
+		Config: Config{
+			ClusterSelector: NetworkPluginKubenetSelector,
+			ClusterMutator:  NetworkPluginKubenetMutator,
+			VHDSelector:     t.AzureLinuxV2Gen2,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-azurelinux-v2-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-azurelinux-v2-gen2"
+			},
+			LiveVMValidators: []*LiveVMValidator{
```

Comment:
can we combine these scenarios/validators?

Reply:
I plan to - once I've managed to get them passing locally. Have been battling issues running the tests against custom VHDs and having them separate until I get the tests passing is useful.

Reply:
Combined now.